packages/cf-style-container/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "fela": "^5.0.0",
+    "fela-utils": "^5.0.2",
```

Comment:
The readme for fela-utils says "fela-utils is a private utility package for Fela and should only be used interally."